nixpkgs_review/cli/__init__.py

```diff
@@ -156,6 +156,20 @@ def common_flags() -> List[CommonFlag]:
             type=regex_type,
             help="Regular expression that package attributes have to match (can be passed multiple times)",
         ),
+        CommonFlag(
+            "-P",
+            "--skip-package",
```

Comment:
Can you also document these flags in the readme?

Reply:
done